paparazzi-preview-processor/src/main/java/app/cash/paparazzi/preview/processor/PreviewProcessor.kt

```diff
@@ -15,5 +19,52 @@ public class PreviewProcessorProvider : SymbolProcessorProvider {
 public class PreviewProcessor(
   private val environment: SymbolProcessorEnvironment
 ) : SymbolProcessor {
-  override fun process(resolver: Resolver): List<KSAnnotated> = emptyList()
+
+  override fun process(resolver: Resolver): List<KSAnnotated> {
+    // Due to codgen and multi-round processing of ksp
+    // https://kotlinlang.org/docs/ksp-multi-round.html
+    if (resolver.getAllFiles().any {
+        it.fileName.contains("PaparazziPreviews") && it.filePath.contains("ksp")
+      }
+    ) {
```

Comment:
i see.  is there a simply way to tell ksp not to consider the previous round's files in the next round?  if not, then `invoked = true` was cleaner imo.

Reply:
yeah I agree 100% I need to look into this.